.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "decco"]
```

Comment:
Isn't there any other ways? I have always experienced a pain while dealing with submodules.

Reply:
Still looking for one :-(

Reply:
Same comment as @AlexITC.

I can see this as an intermediate solution until the `decco` artifact is published somewhere, is that the idea?

Reply:
Yes, that's the idea.